advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/limitations.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Limitations"
+description: Revise any unsupported Oracle data types and features.  
+---
+
+A limited number of Oracle data types and features aren't supported by EDB Data Migration Service (EDB DMS).
+
+See the [Debezium documentation](https://debezium.io/documentation/reference/2.2/connectors/oracle.html#oracle-data-type-mappings) for detailed comments on supported data types. 
+
+Unsupported Oracle data types include:
+
+- BFILE
+- LONG
+- LONG RAW
+- RAW
+- UROWID
+- User-defined types (REF, Varrays, Nested Tables)
+- ANY
+- XML
+- Spatial
+
+EDB DMS supports replicating Oracle tables that contain BLOB, CLOB, or NCLOB columns only if these also have the primary key constraint. If the tables don't have the primary key constraint, the streaming replication will only support INSERT operations.
+
+`BINARY_FLOAT` and `BINARY_DOUBLE` types in Oracle that might contain `Nan`, `+INF`, and `-INF` values are not supported by EDB DMS. 
```

Comment:
A general known issue/limitation was uncovered during my recent testing.  The DMS does not currently support migrating schemas, tables, and columns with mixed case names.  @zahid75 can you confirm that this issue applies to both Oracle and Postgres sources?

A Postgres specific issue was also uncovered.  The DMS does not currently support migrating tables with columns that are defined with certain Domain types. @zahid75 can you provide a list?

@gvasquezvargas we need to add these known current issues/limitations to the documentation here. Since it appears that we have at least one general issue (applies to both Oracle and Postgres) and one Postgres specific issue in addition to the Oracle limitations currently documented here, would it make sense to organize this page with three separate sections: General, Oracle, and Postgres?

Reply:
Yes, both to Oracle and PG. Shared the list of data types with @mlewandowski-edb not covered for DOMAIN type context.

Reply:
@gvasquezvargas @zahid75 confirmed via a slack DM that the mixed case schema, table, or column name issue is applicable to both Oracle and Postgres sources.

Also, regarding the domain type issue, @zahid75 provided the list of types that are not supported.  So, for the new **Postgres** section, we'll need to document the following limitation:
>The EDB DMS does not currently support migrating tables that have columns defined with user defined domains as data types for domains based on the following data types:
> - DATE
> - TIME
> - TIMESTAMP
> - INTERVAL
> - UUID
> - ENUM
> - JSON
> - XML
> - POINT
> - LTREE

In addition to the above limitation, I would also like to include the following issue:
> The The `postgresConfigValidation.sh` script referenced on the Preparing Postgres database sources page [with link to page] incorrectly reports failures for the `max_wal_size` database parameter check and for the replication slot check.  The script will report a failure for the `max_wal_size` check, if the parameter is set to a value lower than 8 GB. Although this is a recommended setting for many production work loads, setting the parameter to a lower value will not prevent a migration from being performed.  In addition, the Postgres configuration validation script also performs a check to see if a replication slot with a slot name matching the migration user/role has been created. This is an old check that is no longer needed as the cdc-reader software automatically creates and manages the needed replication slot. The failure message related to the replication slot can be ignored.  The check will be removed in a future version of the `postgresConfigValidation.sh`script.